modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/throttling/NewThrottleTiersAndLoadOldTiersTestCase.java

```diff
@@ -0,0 +1,610 @@
+
+    private static final Log log = LogFactory.getLog(NewThrottleTiersAndLoadOldTiersTestCase.class);
+    private final String API_TAGS = "testTag1, testTag2, testTag3";
+    private final String API_DESCRIPTION = "This is test API create by API manager integration test";
+    private final String API_VERSION_1_0_0 = "1.0.0";
+    private final String TIER_XML_REG_CONFIG_LOCATION = "/_system/governance/apimgt/applicationdata/tiers.xml";
+    private final String TIER_XML_REG_CONFIG_APP_LOCATION = "/_system/governance/apimgt/applicationdata/app-tiers.xml";
+    private final String TIER_XML_REG_CONFIG_RES_LOCATION = "/_system/governance/apimgt/applicationdata/res-tiers.xml";
+    private final String TIER_MANAGE_PAGE_TIER_GOLD = "{ \"value\": \"Gold\", \"text\": \"Gold\" }";
+    private final String TIER_MANAGE_PAGE_TIER_SILVER = "<option value=\"Silver\" >Silver</option>";
+
+    private final String API_END_POINT_POSTFIX_URL = "jaxrs_basic/services/customers/customerservice/";
+    private final String TIER_MANAGE_PAGE_RESOURCE_TIER_ULTIMATE = "\"value\": \"Ultimate\"";
+    private final String TIER_MANAGE_PAGE_RESOURCE_TIER_SILVER = "<option value=\"Silver\" >Silver</option>";
+    private final String TIER_MANAGE_PAGE_APPLICATION_TIER_GOLD = "<option value=\"Gold\" >";
+    private final String TIER_MANAGE_PAGE_APPLICATION_TIER_LARGE = "<option value=\"Large\" >";
+
+    private final int HTTP_RESPONSE_CODE_OK = Response.Status.OK.getStatusCode();
+    private final int HTTP_RESPONSE_CODE_TOO_MANY_REQUESTS = 429;
+
+    private final String NEW_TIER_TEST_API_CONTEXT = "new_Tiers_api";
+    private final String New_Tier_TEST_API = "new_Tiers_api_context";
+
+    private final String API_TIER_TEST_API = "test_APITier_api";
+    private final String API_TIER_TEST_APP = "test_APITier_app";
+    private final String API_TIER_TEST_API_CONTEXT = "test_APITier_api_context";
+
+    private final String APP_TIER_TEST_API = "test_APPTier_api";
+    private final String APP_TIER_TEST_API_CONTEXT = "test_APPTier_api_context";
+    private final String APP_TIER_TEST_APP = "test_APPTier_app";
+
+    private final String RES_TIER_TEST_API = "test_RESTier_api";
+    private final String RES_TIER_TEST_API_CONTEXT = "test_RESTier_api_context";
+    private final String RES_TIER_TEST_APP = "test_RESTier_app_name";
+
+    private final String OLD_TIER_TEST_API = "old_Tier_api";
+    private final String OLD_TIER_TEST_API_CONTEXT = "old_Tier_api_context";
+
+    private String apiEndPointUrl;
+    private String providerName;
+    private String newTiersXML;
+    private ResourceAdminServiceClient resourceAdminServiceClient;
+    private APIPublisherRestClient apiPublisherClientUser;
+    private APIStoreRestClient apiStoreClientUser;
+
+    private APIIdentifier APITierTEST_ID;
+    private APIIdentifier APPTierTEST_ID;
+    private APIIdentifier RESTierTEST_ID;
+    private APIIdentifier Before_TierTEST_ID;
+    private APIIdentifier After_TierTEST_ID;
```

Comment:
revert changes already in commit: https://github.com/wso2/product-apim/commit/a75c4880b08d984d9a0e971387c0d6a07ef2c886